index.html

```diff
@@ -504,60 +499,26 @@ <h4 id="return-values">Return Value</h4>
           </dl>
         </li>
         <li>
-          <p>Set <var>age</var> to current time minus <var>requestTime</var>
-          expressed in seconds. If <var>age</var> is non-zero, append a
-          <dfn><code>Beacon-Age</code></dfn> header with value <var>age</var>
-          to <a>header list</a> field of <var>req</var>.</p>
-        </li>
-        <li>
           <a>Fetch</a> <var>req</var>.
         </li>
       </ol>
-      <p>The <dfn>Beacon-Age header field</dfn> is used to communicate the time
-      delay, measured in seconds, between the time when the request is sent and
-      the <i>requestTime</i> set when <a>sendBeacon</a> method is called. The
-      ABNF (Augmented Backus-Naur Form) syntax for the <a>Beacon-Age header
-      field</a> is as follows:</p>
-      <pre>
-Beacon-Age = "Beacon-Age" ":" 1*DIGIT
-</pre>
     </section>
-    <section id="privacy" class='informative'>
+    <section id="privacy" class="informative">
       <h2>Privacy and Security</h2>
-      <p>The <code>sendBeacon</code> interface provides an asynchronous and
-      non-blocking mechanism for delivery of data. It is designed to eliminate
-      the need for the widespread use of expensive anti-patterns that block
-      user interaction, and to enable more efficient scheduling and use of
-      system resources. The same functionality is possible today, albeit with
-      high performance costs and degraged user experience.</p>
-      <p><code>sendBeacon</code> does not add new privacy considerations. However, implementers should take the following precautions:</p>
+      <p>The <code>sendBeacon</code> interface provides an asynchronous and non-blocking mechanism for delivery of data. This API may be used to:</p>
```

Comment:
You probably want to use "can be" here since "may" is a normative term and makes little sense to use here.